pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -86,6 +86,15 @@ type PrometheusSpec struct {
 
 	// +optional
 	Ingress *IngressSpec `json:"ingress,omitempty"`
+
+	Container *corev1.Container `json:"container,omitempty"`
```

Comment:
It seems we only want to expose start command to user. I recommended adding `Commands`  in `Configuration` instead of adding `Container` in `PrometheusSpec`.